zapisy/apps/enrollment/courses/views.py

```diff
@@ -256,10 +266,18 @@ def group_view(request, group_id):
     )
     group: Group = enrolled_data[group_id].get("group")
 
+    limits_for_roles = [gs.limit for gs in GuaranteedSpots.objects.filter(group=group)]
+    group.total_limit = group.limit + sum(limits_for_roles)
+
+    enrolled = Record.taken_spots_by_role([group])
+    group.num_enrolled = enrolled[group.pk].pop("")
```

Comment:
To się posypie, jeśli wszyscy zapisani do grupy będą mieli jakieś role spośród tych, które są do niej przypisane (najbardziej typowe jest to wtedy, kiedy wszystkie miejsca są "zarezerwowane", ale to nawet nie jest warunek konieczny). Oczywiście _hotfix_ to
```suggestion
    group.num_enrolled = enrolled[group.pk].pop("", 0)
```
ale jeszcze nie zastanowiłem się, jak to wpływa na logikę w innych miejscach.